tests/unit/utils/sparqlhandler/test_sparqlmanager.py

```diff
@@ -159,32 +241,203 @@ def test_select_csvw_catalog_metadata_for_codelist():
     assert result.identifier == "Alcohol Content"
 
 
-def test_select_csvw_dsd_dataset():
+def test_select_csvw_dsd_dataset_for_pivoted_multi_measure_data_set():
```

Comment:
Do we need to test this for the old-style single measure pivoted input? The inspect functionality should still be capable of querying and interpreting it, even if the corresponding version of csvcubed can't output it anymore.